#2817: Pique: Overrides paragraph block alignment added on cover block on home page

Gustavo-Hilario opened the pull request:
Ensuring paragraph alignment is preserved on hero banners on `Pique` home page.

#### Changes proposed in this Pull Request:

Fixing alignment issues on hero banner on `Pique` home page.

<table>
    <thead>
      <tr>
        <th>Before</th>
        <th>After</th>
    </tr>
    </thead>
    <tbody>
      <tr>
        <td><img src="https://user-images.githubusercontent.com/50875131/180620488-d10c98e9-4df8-47c2-b45a-b8b1045923d0.png"/></td>
        <td><img src="https://user-images.githubusercontent.com/50875131/180620497-2031ca16-a249-4f5c-b7b0-fd64d520311f.png"/></td>
      </tr>
    </tbody>
  </table>

#### Related issue(s):

Pique Theme: Overrides paragraph block alignment added on cover block on home page #2817

Gustavo-Hilario committed:
#2817: Pique: Overrides paragraph block alignment added on cover block on home page

Ensuring paragraphs alignment is preserved on hero banners on Pique panels -- home page

madhusudhand commented on pique/style.css:
```diff
@@ -2575,7 +2575,7 @@ body:not(.no-background-fixed) .site-footer {
 	text-align: center;
 }
-#pique-hero .pique-panel-content .entry-content p {
+#pique-hero .pique-panel-content .entry-content p:not(*) {
```
```suggestion
:where(#pique-hero .pique-panel-content .entry-content) p {
```

This code changes the default paragraph alignment to `center` where as in the normal page it says left aligned. Is that expected?

<img width="322" alt="image" src="https://user-images.githubusercontent.com/1935113/190135644-35e259b8-3ecd-47af-b341-8444e8b68e71.png">

I suggest to lower the specificity using [:where](https://developer.mozilla.org/en-US/docs/Web/CSS/:where).

Gustavo-Hilario replied:
Yeah. I didn't notice that code was changing the alignment for default paragraph blocks. Thank you for the heads up!

Gustavo-Hilario committed:
Fixing paragraph alignment issue on Pique home page panel

A correction will ensure that paragraphs with the default alignment will be left aligned on the Pique home page.

Co-authored-by: Madhu Dollu <madhusudhan.dollu@gmail.com>

madhusudhand reviewed (approved):
LGTM!
